requirements.txt

```diff
@@ -1,4 +1,3 @@
--c constraints-deps.txt
 -c constraints.txt
 APScheduler
 cached-property
```

Comment:
It appears that `sqlalchemy-postgres-copy` has a hard dependency on `psyopg2` and installs it in addition to `psycopg2-binary`.

PR here to resolve that: https://github.com/jmcarp/sqlalchemy-postgres-copy/pull/14